src/getBlobFromURL.ts

```diff
@@ -10,10 +10,14 @@ const cache: {
   [url: string]: Promise<Metadata>
 } = {}
 
-function getCacheKey(url: string) {
+function getCacheKey(url: string, includeQueryParams: boolean | undefined) {
   let key = url.replace(/\?.*/, '')
```

Comment:
Nitpick: No need to replace if we will reassign the full url after so we should replace only if it's not to includeQueyParams right?